app/models/parliament.rb

```diff
@@ -33,8 +37,14 @@ def last_or_create
     end
   end
 
+  validates_presence_of :dissolution_heading, :dissolution_message, if: :dissolution_at?
+  validates_length_of :dissolution_heading, maximum: 100
+  validates_length_of :dissolution_message, maximum: 600
+
+  after_save { Site.touch }
```

Comment:
Is this so that we bust the petition fragment cache? If so are we intentionally busting that when the site gets updated or could we use the parliament `updated_at` timestamp?